xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4440,7 +4440,11 @@ bool CVideoPlayer::OnAction(const CAction &action)
     }
   }
 
-  if (dynamic_cast<CDVDInputStreamPVRManager*>(m_pInputStream))
+  //If the current channel is a nullptr, and this is a PVRStream, then this is a saved recording
+  CDVDInputStreamPVRManager* pvrStream = dynamic_cast<CDVDInputStreamPVRManager*>(m_pInputStream);
```

Comment:
Revert this +/-, please. <code>pvrStream</code> variable is no longer needed (and the comment can be removed as well, imo).